mkdocs.yml

```diff
@@ -0,0 +1,69 @@
+---
+nav:
+    - Home: index.md
+    - Installation: installation/index.md
+    - Getting Started:
+          - 1. Initial Steps: getting-started/initial-steps.md
+          - 2. Create a Component: getting-started/create-component.md
+          - 3. Use the Template Tag: getting-started/reference-component.md
+          - 4. Render Your View: getting-started/render-view.md
+          - 5. Learn More: getting-started/learn-more.md
+    - Exclusive Features:
+          - Hooks: features/hooks.md
+          - Template Tag: features/templatetag.md
+    - Contribute:
```

Comment:
Maybe add a section that links to contribution docs for the core project?

Reply:
I added a `tip` to the page that links to IDOM-Core. Let me know what you think.